File: app/src/main/java/org/mifos/mobilebanking/models/AccountOptionAndBeneficiary.java
package org.mifos.mobilebanking.models;

import org.mifos.mobilebanking.models.beneficary.Beneficiary;
import org.mifos.mobilebanking.models.templates.account.AccountOptionsTemplate;

import java.util.List;

/**
 * Created by dilpreet on 23/6/17.
 */

public class AccountOptionAndBeneficiary {

    private AccountOptionsTemplate accountOptionsTemplate;
    private List<Beneficiary> beneficiaryList;

    public AccountOptionAndBeneficiary(AccountOptionsTemplate accountOptionsTemplate,
                                       List<Beneficiary> beneficiaryList) {
        this.accountOptionsTemplate = accountOptionsTemplate;
        this.beneficiaryList = beneficiaryList;
    }

    public List<Beneficiary> getBeneficiaryList() {
        return beneficiaryList;
    }

    public AccountOptionsTemplate getAccountOptionsTemplate() {

        return accountOptionsTemplate;
    }
}

File: app/src/main/java/org/mifos/mobilebanking/models/AccountOptionAndBeneficiary.kt
package org.mifos.mobilebanking.models

import org.mifos.mobilebanking.models.beneficiary.Beneficiary
import org.mifos.mobilebanking.models.templates.account.AccountOptionsTemplate

/**
 * Created by dilpreet on 23/6/17.
 */

data class AccountOptionAndBeneficiary(
        val accountOptionsTemplate: AccountOptionsTemplate,
        val beneficiaryList: List<Beneficiary>)
